frontend/mocking/mocker.js

```diff
@@ -554,53 +587,130 @@ const schemaWithMocks = addMocksToSchema({
       },
     },
     Mutation: {
+      refreshTokens: (_, _args, context, _resolveInfo) => {
+        if (!context.cookies.refresh_token) {
+          return {
+            result: {
+              code: 401,
+              description: 'Failed to refresh tokens. Please log in.',
+              type: 'AuthenticateError',
+            },
+          }
+        }
+
+        const cookieData = {
+          maxAge: 1000 * REFRESH_TOKEN_EXPIRY_SECONDS,
+          httpOnly: true,
+          secure: false,
```

Comment:
refresh tokens stored with secure: true
```suggestion
          secure: true,
```